.circleci/src/commands/notify.yml

```diff
@@ -13,7 +13,7 @@ parameters:
     default: main
     type: string
   channel:
-    default: integreat-app-notifications
+    default: releases
```

Comment:
:x: Please revert this, this will change the channel for ALL notifications, not just for promotions. Have a look at `notify_android.yml` for example on how to use the `notify` to a different channel.